pkg/cluster/docker_for_mac.go

```diff
@@ -0,0 +1,199 @@
+package cluster
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net"
+	"net/http"
+	"path/filepath"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/pkg/errors"
+	klog "k8s.io/klog/v2"
+)
+
+type d4mSettings map[string]interface{}
+
+type HTTPClient interface {
+	Do(req *http.Request) (*http.Response, error)
+}
+
+// Uses the DockerForMac GUI protocol to control DockerForMac.
```

Comment:
Do you have a reference you can link to here?

Reply:
there's not a public one, see: https://app.clubhouse.io/windmill/story/9899/add-the-ability-to-set-the-cpu-of-docker-for-desktop

Reply:
😢  whelp. Good find though!

I think you should mention that in a comment/readme/something so that someone trying to make changes to this client has some footing.